<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalendarViews Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="calendarView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="addViewMenu">
    <property name="title">
     <string>Add View</string>
    </property>
    <addaction name="actionAgenda"/>
    <addaction name="actionMultiAgenda"/>
    <addaction name="actionMonth"/>
    <addaction name="actionTimeline"/>
   </widget>
   <addaction name="addViewMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAgenda">
   <property name="text">
    <string>Agenda</string>
   </property>
  </action>
  <action name="actionMultiAgenda">
   <property name="text">
    <string>MultiAgenda</string>
   </property>
  </action>
  <action name="actionMonth">
   <property name="text">
    <string>Month</string>
   </property>
  </action>
  <action name="actionTimeline">
   <property name="text">
    <string>Timeline</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
